sympy/functions/special/tests/test_error_functions.py

```diff
@@ -235,20 +275,31 @@ def test_erf2():
     assert erf2(x, y).rewrite('erfc') == erfc(x) - erfc(y)
     assert erf2(x, y).rewrite('erfi') == I*(erfi(I*x) - erfi(I*y))
 
-    raises(ArgumentIndexError, lambda: erfi(x).fdiff(3))
+    assert erf2(x, y).diff(x) == erf2(x, y).fdiff(1)
+    assert erf2(x, y).diff(y) == erf2(x, y).fdiff(2)
+    assert erf2(x, y).diff(x) == -2*exp(-x**2)/sqrt(pi)
+    assert erf2(x, y).diff(y) == 2*exp(-y**2)/sqrt(pi)
+    raises(ArgumentIndexError, lambda: erf2(x, y).fdiff(3))
+
+    assert erf2(x, y).is_extended_real is None
+    xr, yr = symbols('xr yr', extended_real=True)
+    assert erf2(xr, yr).is_extended_real is True
```

Comment:
Should this also work for real=True?

Reply:
Not sure, but I think so. (It means that x, y < oo gives erf2 < oo, right?)

Reply:
With a sloppy formulation, forgetting about negative numbers etc...

Reply:
Yes, if x and y are finite then I think erf2(x, y) is finite. So if x and y are declared as real then erf2(x, y).is_real should give True.

However this doesn't work right now and erf2(x, y).is_real and is_finite both give None. Adding `_eval_is_finite` would make it work for both. I think that the error function is finite whenever both  x and y are extended real or when they are both finite. It has poles at `+-I*oo` in either argument I think...